delta/migration/src/test/resources/resource.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
It is migration code so it is ok, it will be deleted anyway soon but for this kind of payload, we can use `circe-literal`